### ISSUE TITLE
unable to toggle entire groups

### ISSUE THREAD

**Issue by vdmkenny** [MycroftAI/skill-homeassistant#19]
I have multiple rooms configured in my home-assistant, but I'm unable to shut off lights on a per-room basis.

When I try the utterance "turn off the living room lights", and see it correctly registered in the logs, it still matches a scene I have configured to turn my living room lights on low or high.

The reply I get is "turn off living room nighttime", but my lights stay on.

I can ofcourse ask to turn all my lights in the house off, but this defeats the point. I'd rather not create a scene with shut off lights if I can help it.

Same goes for my kitchen, the utterance "turn on/off kitchen lights", and it's correctly interpreted in the logs, but yet, it goes to turn on "all my switches", so my 3d printer in the next room turns on instead.

Am I using the wrong utterances? It seems like mycroft's confidence is way off and it keeps going for the wrong things.

In short, how do I address an entire group/room at once without a scene?

**Comment by vdmkenny** [comment]
It seems like I misunderstood something about home-assistant.
When my lights are discovered and placed into rooms in the UI, this does not correspond to groups.
When I manually put them in groups, it works. Sorry about the confusion.
